tutor/src/components/task-plan/index.cjsx

```diff
@@ -56,9 +58,13 @@ EventShell = React.createClass
 PlanShell = React.createClass
   displayName: 'PlanShell'
 
```

Comment:
hm, yeah that'd be good. I think the old Router used to do that, right?  We'll have to make it so that it can be overridden though, since some routes won't want that, like the metrics review used to